controllers/doc.go

```diff
@@ -14,27 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package controller
-
-import (
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-)
-
 // +kubebuilder:rbac:groups=core,resources=events,verbs=get;list;watch;create;patch
 // +kubebuilder:rbac:groups=core,resources=secrets,verbs=get;list;watch
 // +kubebuilder:rbac:groups=core,resources=nodes,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=clusters;clusters/status;machines;machines/status;machinedeployments;machinedeployments/status;machinesets;machinesets/status,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=infrastructure.cluster.x-k8s.io;bootstrap.cluster.x-k8s.io,resources=*,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Can these be moved to the individual controllers where they are needed? It should be fine to duplicate, since the generation tooling should dedup.

Reply:
Gonna open an issue to clean this up

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/1272